packages/core/e2e/e2e.test.ts

```diff
@@ -579,14 +582,15 @@ describe('e2e', () => {
       expect(returnValue.cause).toHaveProperty('stack');
       expect(typeof returnValue.cause.stack).toBe('string');
 
-      // Known issue: SvelteKit dev mode has incorrect source map mappings for bundled imports.
+      // Known issue: vite-based frameworks dev mode has incorrect source map mappings for bundled imports.
       // esbuild with bundle:true inlines helpers.ts but source maps incorrectly map to 99_e2e.ts
       // This works correctly in production and other frameworks.
       // TODO: Investigate esbuild source map generation for bundled modules
-      const isSvelteKitDevMode =
-        process.env.APP_NAME === 'sveltekit' && isLocalDeployment();
+      const isViteBasedFrameworkDevMode =
+        process.env.APP_NAME === 'sveltekit' ||
```

Comment:
this should also be inside the `devmode` check. currently you're diabling this test for sveltekit on ALL environments, but it was working in other envs I believe